packages/core/logging/core-logging-browser-internal/src/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { LogLevelId } from '@kbn/logging';
+
+/**
+ * Describes the configuration of a given logger.
+ *
+ * @public
+ */
+export interface LoggerConfigType {
+  appenders: string[];
+  name: string;
+  level: LogLevelId;
+}
```

Comment:
This is a duplication from the type in the `@kbn/core-logging-server` package (`packages/core/logging/core-logging-server/src/logger.ts`).

When we'll eventually want to support full browser-side logging configuration, we will need to move all the logging config types from the `@kbn/core-logging-server` to a common one, but given this wasn't necessary for the first implementation, I simply copied the single type I'm using.

Reply:
I created https://github.com/elastic/kibana/issues/144276 as a way to track enhancements and priority for further work in this area.